fme/requirements.txt

```diff
@@ -6,10 +6,10 @@ numpy<2
 wandb
 tensorly
 tensorly-torch
-xarray
+xarray>=2023.2.0
```

Comment:
Mostly out of curiosity.. was there a particular feature that required this version?

Reply:
With the original setup.py `python>=3.8` the tests were failing without xarray above that version (which also forced the bump to `python>=3.9`).  I removed since the python requirement automatically solves the xarray version correctly.

Reply:
Thanks, that is helpful explanation. I'm +1 on requiring python 3.9 given EOL for 3.8.